components/cluster/cluster-glue/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright (c) 2013 Adam Stevko <adam.stevko@gmail.com>
+# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= cluster-glue
+COMPONENT_VERSION= 1.0.12
+COMPONENT_FMRI= application/cluster/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= System/Services
+COMPONENT_HG_REV= 0a7add1d9996
+COMPONENT_SRC= Reusable-Cluster-Components-glue--$(COMPONENT_HG_REV)
+COMPONENT_ARCHIVE= $(COMPONENT_HG_REV).tar.bz2
```

Comment:
May be I misunderstand the comments, but it is not my setup, its is the reality.
I understand the problem but I have not clue how to solve it.
Then url and the name of the archive are defined by the authors, see here:
http://linux-ha.org/wiki/Downloads
Problems arise with 
Cluster Glue 1.0.12: http://hg.linux-ha.org/glue/archive/0a7add1d9996.tar.bz2
Resource Agents 3.9.6: https://github.com/ClusterLabs/resource-agents/archive/v3.9.6.tar.gz
CRM Shell 2.1.2: https://github.com/crmsh/crmsh/archive/2.1.2.tar.gz 
In order to transform this ugly namings into user friendly archive names suitable to be stored in a common download directory, the archives mus be renamed after download.
Is this possible?